common/app/views/fragments/headerTopNav.scala.html

```diff
@@ -0,0 +1,206 @@
+                    Header) else getReaderRevenueUrl(PrintCTAWeekly, Header)) { href =>
+                    <a class="top-bar__item hide-until-desktop yellow" data-link-name="nav3 : print-cta"
+                        data-edition="@{editionId}" href="@href">
+                        @fragments.inlineSvg("newspaper", "icon", List("top-bar__item__icon"))
+                        Print subscriptions
+                    </a>
+                    }
+                </div>
+
+
+
+                <div class="header-top-nav__item">
+                    <span class="header-top-nav__item--separator hide-until-desktop"></span>
+                    <a class="top-bar__item js-navigation-sign-in my-account" data-link-name="nav3 : topbar : signin"
+                        href="@Configuration.id.url/signin?INTCMP=DOTCOM_NEWHEADER_SIGNIN&@createAuthenticationComponentEventParams(SigninHeaderId)">
+                        @fragments.inlineSvg("profile", "icon", List("top-bar__item__icon"))
+                        Sign in
+                    </a>
+
+                    <input type="checkbox" id="my-account-toggle" aria-controls="my-account-dropdown"
+                        class="dropdown-menu-fallback js-enhance-checkbox" data-link-name="nav3 : topbar: my account"
+                        tabindex="-1" />
+
+                    <label
+                        class="top-bar__item popup__toggle js-navigation-account-actions js-user-account-trigger is-hidden"
+                        for="my-account-toggle" data-link-name="nav3 : topbar: my account" tabindex="0">
+                        @fragments.inlineSvg("profile", "icon", List("top-bar__item__icon"))
+                        My account
+                    </label>
+                    <div class="my-account__overlay"></div>
+
+                    <ul class="dropdown-menu dropdown-menu--light js-user-account-dropdown-menu"
+                        id="my-account-dropdown" aria-hidden="true">
+
+                        @for((item)
+                        <- accountDropdownMenu) { @if(item.divider) { <hr />
+                        }
+                        <li class="@{(List(" dropdown-menu__item") ++ item.parentClassList).mkString(" ")}">
+                            <a class="@{(List(" dropdown-menu__title") ++ item.classList).mkString(" ")}"
+                                @if(item.href.isDefined) { href="@item.href" } @if(item.linkName.isDefined) {
+                                data-link-name="nav3 : topbar : @item.linkName" }>
+                                @(item.icon.map { icon =>
+                                views.html.fragments.inlineSvg(icon, "icon", isPresentation = true)
+                                })
+                                @item.label
+                            </a>
+                        </li>
+                        }
+                    </ul>
+                </div>
+
+                @if(editionId != "au" || editionId == "uk") {
```

Comment:
```suggestion
                @if(editionId != "au") {
```
Surely if `editionId` was `uk` it would already be caught by the `editionID != "au"`?

Reply:
This is how it was… but I see your point.